dist/common/scripts/scylla_setup

```diff
@@ -193,30 +193,30 @@ if __name__ == '__main__':
     default_no_coredump_setup = False
     default_no_cpuscaling_setup = False
 
-    if is_offline():
-        if not is_nonroot():
-            default_no_ntp_setup = True
-            warn_offline('ntp setup')
-            if not shutil.which('mkfs.xfs'):
-                default_no_kernel_check = True
-                default_no_raid_setup = True
-                warn_offline_missing_pkg('kernel version check', 'xfsprogs')
-                warn_offline_missing_pkg('RAID setup', 'xfsprogs')
-
-            if not default_no_raid_setup and not shutil.which('mdadm'):
-                default_no_raid_setup = True
-                warn_offline_missing_pkg('RAID setup', 'mdadm')
-
-            if not shutil.which('coredumpctl'):
-                default_no_coredump_setup = True
-                warn_offline_missing_pkg('coredump setup', 'systemd-coredump')
-
-            if is_debian_variant() and not shutil.which('cpufreq-set'):
-                default_no_cpuscaling_setup = True
-                warn_offline_missing_pkg('cpuscaling setup', 'cpufrequtils')
-            elif not shutil.which('cpupower'):
-                default_no_cpuscaling_setup = True
-                warn_offline_missing_pkg('cpuscaling setup', 'cpupower')
+    if not is_nonroot():
```

Comment:
1. There is no description why we need to remove `if is_offline():`
2. If we remove `if is_offline():`, normal package users (who installed scylla via apt / yum) will see these offline mode messages, it must be wrong.

Reply:
Fixed.